WalletWasabi/Services/WalletService.cs

```diff
@@ -753,7 +753,7 @@ private async Task<Block> TryDownloadBlockFromLocalNodeAsync(uint256 hash, Cance
 						{
 							ConnectCancellation = handshakeTimeout.Token,
 							IsRelay = false,
-							UserAgent = $"/Wasabi:{Constants.ClientVersion.ToVersionString()}/"
+							UserAgent = $"/Wasabi:{Constants.ClientVersion.ToString(3)}/"
```

Comment:
@lontivero is this Ok?

Reply:
That should be fine

Reply:
This part does not really matter, it just changes what appears on your local node when wasabi connects to it

Reply:
Yes, the change is okay.